### PR TITLE
NYC EITC

### LINKED ISSUE #1281: NYC EITC
5% of federal
https://www.tax.ny.gov/pdf/current_forms/it/it215i.pdf
https://codelibrary.amlegal.com/codes/newyorkcity/latest/NYCadmin/0-0-0-13608

### PR DESCRIPTION
**Pull request by leogoldman**
NYC EITC
Fixes #1281

### CODE REVIEW

File: policyengine_us/variables/gov/local/ny/nyc/tax/income/credits/nyc_eitc.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+        # First get their NYAGI and EITC.
+        ny_agi = tax_unit("ny_agi", period)
+        federal_eitc = tax_unit("earned_income_tax_credit", period)
+
+        # Then get the EITC part of the parameter tree.
+        p = parameters(period).gov.local.ny.nyc.tax.income.credits.eitc
+
+        # Calculate the EITC before the reduction.
+        percentage_pre_reduction = p.percentage.calc(ny_agi)
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/local/ny/nyc/tax/income/credits/nyc_eitc.py:22]: Changing to percent will fix the test

**Comment by MaxGhenis** [comment]
This plus a make format should do it

**Review by leogoldman — COMMENTED** [review]
Changed "percentage" to "percent"

**Comment by MaxGhenis** [comment]
Thank you @leogoldman and congrats on your first merged PR in policyengine-us!

**Pull request merged**